pkg/instance/instance_integration_test.go

```diff
@@ -207,6 +207,46 @@ func TestInstanceHandler(t *testing.T) {
 		time.Sleep(3 * time.Second)
 		k8sClient.AssertPodIsNotRunning(t, deploymentInstance.GroupName, deploymentInstance.Name)
 	})
+
+	t.Run("GetPublicDeployments", func(t *testing.T) {
+		t.Log("Create deployment")
+		var deployment model.Deployment
+		body := strings.NewReader(`{
+			"name": "private-deployment",
+			"group": "group-name",
+			"description": "some description"
+		}`)
+		client.PostJSON(t, "/deployments", body, &deployment, inttest.WithAuthToken("sometoken"))
```

Comment:
Please add newlines in between the setup, the thing we test and the assertions as we do in other tests.

```
setup

client.Post..

assert
```